Path-Check/gaen-mobile
# Reference specific commit from pathcheck-mobile-resources

#### Description:


We need a way to check which configuration we are using for each build.
Instead of always getting the latest changes from `master` this PR allows us to target a specific commit.

Each time there's a configuration change in https://github.com/Path-Check/pathcheck-mobile-resources we will have to update the file `/bin/mobile-resources-commit` to reflect those changes in the app.

I also allows us to define a different configuration per branch (for `mn-release` branch for example)

#### Linked issues:


https://trello.com/c/RZKjDdRu/404-as-a-dev-team-we-have-a-way-to-version-a-build-to-the-correct-commit-to-the-pathcheck-mobile-resources-repo

#### How to test:


Run `./bin/set_ha.sh mn` to check that it reads the configuration from the commit specified in `/bin/mobile-resources-commit`
